trillian_admin_api.proto

```diff
@@ -20,7 +20,7 @@ option java_outer_classname = "TrillianApiProto";
 
 package trillian;
 
-import "trillian.proto";
+import "github.com/google/trillian/trillian.proto";
```

Comment:
- I like the idea of putting all the proto files under `trillian/proto`
- Typically pb.go files are placed in the same directory as their proto files. This makes it obvious which gen.go needs to be invoked to update it. Is there a different pattern that you're seeing? 
- What import path do you suggest that external projects use? Do we want to require relying projects to add `-I=$GOPATH/github.com/google/trillian/proto` to their protoc invocations?